src/main/java/com/philips/research/spdxbuilder/persistence/blackduck/BlackDuckApi.java

```diff
@@ -14,10 +14,7 @@
 
 import java.net.URI;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
explicit imports are better imho.

Reply:
if 5 or more imports from same package, we have a code style rule to replace it with a asterik. Its a default. I see the same behavior in other classes as well,